windows/helpers/update_root_certs.ps1

```diff
@@ -8,14 +8,15 @@ function Invoke-WithRetry {
         Runs $command block until $BreakCondition or $RetryCount is reached.
      #>
 
-     param([ScriptBlock]$Command, [ScriptBlock] $BreakCondition, [int] $RetryCount=5, [int] $Sleep=10)
+     param([ScriptBlock]$Command, [ScriptBlock] $BreakCondition, [int] $RetryCount=10, [int] $Sleep=60)
```

Comment:
I'd propose reducing the sleep interval and bumping retry count, e.g. sleep 30 and retry count 20. It doesn't cost us anything to do more retries, but making the build faster by potentially succeeding 30 seconds earlier would be great.

Reply:
Yes I do not have any problem with this! I'll make the change